src/pages/blog/BlogPage.tsx

```diff
@@ -40,7 +40,8 @@ export default function BlogPage() {
             >
               {repeatComponent(<PostCardSkeleton />, postSkeletonCount)}
             </Masonry>
-          ) : data?.posts.length ? (
+          )}
+          {!loading && data?.posts.length && data.posts.length > 0 && (
```

Comment:
Avoid using complex conditions inside jsx. Move it to variable `shouldRenderMasonry`

Reply:
Fixed

![image](https://github.com/user-attachments/assets/ad06a7cb-76a3-4e8d-a116-f8bc8e1b8a36)